core/src/main/java/io/questdb/std/IntStack.java

```diff
@@ -72,13 +72,16 @@ public int pop() {
     public int pollLast() {
         final int[] es = elements;
         final int t;
-        int e = es[t = dec(tail, es.length)];
+        final int e = es[t = dec(tail, es.length)];
         tail = t;
+        es[t] = noEntryValue;
         return e;
     }
 
-    static int dec(int i, int modulus) {
-        if (--i < 0) i = modulus - 1;
+    private  int dec(int i, int modulus) {
```

Comment:
nit: extra space here.

Reply:
i'm paranoid about formatting code because it reshuffles methods, not used to count spaces :(

Reply:
removed: https://github.com/questdb/questdb/pull/2037/commits/f38e19731353ed9d2bf6ce4422741f04eabe7873